CHANGELOG.md

```diff
@@ -4,8 +4,8 @@ x.y.z Release notes (yyyy-MM-dd)
 * None.
 
 ### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-cocoa/issues/????), since v?.?.?)
-* None.
+
+* Fixed a crash that would occur when observing unmanaged Objects in multiple views in SwiftUI. We now correctly remove KVOs only once.
```

Comment:
This is still pretty unclear. It crashed when using ObservedRealmObject (I think?), not just Realm with SwiftUI in general. It also needs to link to the issue.